echoregions/regions2d/regions2d.py

```diff
@@ -501,3 +501,226 @@ def mask(
             M = M.rename(mask_var)
 
         return M
+
+    def transect_mask(
+        self,
+        da_Sv: DataArray,
+        transect_dict: dict = {
+            "start": "ST",
+            "break": "BT",
+            "resume": "RT",
+            "end": "ET",
+        },
+        file_names: List[str] = None,
```

Comment:
Which filenames does this refer to: the zarr filenames or the evr filenames? I guess if the r2d object is coming from one file there is no need for this. Maybe for now the approach would be to match to the Sv and that is it.

Reply:
Yeah, I'm just removing this. It referred to the Zarr filenames.